scripts/run_commands.sh

```diff
@@ -0,0 +1,70 @@
+declare -i FAILURES
+FAILURES=0
+
+RED='\033[1;31m'
+GREEN='\033[1;32m'
+LRED='\033[0;31m'
+LGREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+expect_success() {
+	if "${@}" &>> ~/test_output.txt; then
+    echo -e "${GREEN}SUCCEEDED${NC}: ${LGREEN}${@}${NC}"
+  else
+    echo "-------"
+    echo -e "${RED}FAILED${NC}: ${LRED}${@}${NC}"
+    echo "-------"
+    FAILURES+=1
+  fi
+}
+
+# install git and go
+echo "TYPE y|yes then enter to proceed."
+sudo yum install git go >> ~/test_log.txt
+# have to respond yes to prompt
+# get CLI
+export GOPATH="$HOME/go"
+go get github.com/aws/amazon-ecs-cli >> ~/test_log.txt
+cd go/src/github.com/aws/amazon-ecs-cli
+url="https://github.com/"
+url+=$gitname
```

Comment:
We talked about this a bit off line, but just noting here that on first past, it's not immediately obvious where this and other env variables are being set. Perhaps worth adding a comment or some other kind of documentation on how the test files are supposed to fit together?

Reply:
Sure, I'll add a comment at the top of each file.